Tests currently broken: TypeError: Arguments to path.resolve must be strings

Hello,

At the moment the tests are broken (fresh clone of the master branch + node v0.10.12 + Linux 2.6 x86_64) :

```
github.com/node-ldapjs$ npm test

> ldapjs@0.6.3 test /home/sweet/version/github.com/node-ldapjs
> ./node_modules/.bin/tap ./test


path.js:313
        throw new TypeError('Arguments to path.resolve must be strings');
              ^
TypeError: Arguments to path.resolve must be strings
    at Object.exports.resolve (path.js:313:15)
    at Array.map (native)
    at /home/sweet/version/github.com/node-ldapjs/node_modules/tap/lib/tap-runner.js:119:19
    at Object.oncomplete (fs.js:107:15)
npm ERR! weird error 8
npm ERR! not ok code 0
```

Could that be fixed please?

Best,

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "pooling": "0.4.4"
   },
   "devDependencies": {
-    "tap": "0.4.0",
+    "tap": "0.4.1",
     "node-uuid": "1.4.0"
   },
   "scripts": {

